### DISCUSSION TITLE
Fetch distinct data from the API

### DISCUSSION THREAD

**Discussion by SavArmadillo** [directus/directus#14846] [Archive]
Hello!

I've been wondering if the directus API get the distinct data.

Let's say I have a data table with the following field and data:
```json
{
  "team": ["Jhon Doe", "Alice"]
},
{
  "team": ["Alice", "Bob"]
},
{
  "team": ["Jhon Doe"]
},
```
the field team is either JSON or CSV.
Is it possible, using the api calls, to get a response that looks like:
```json
["Jhon Doe", "Alice", "Bob"]
```

Thanks!

**Comment by rijkvanzanten** [comment]
No, there's currently no way to perform distinct / merging operations on JSON values returned from the API 👍🏻
